ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vxlan-interface.j2

```diff
@@ -4,30 +4,54 @@
 
 ### VXLAN Interface Summary
 
-#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.source_interface }}
+#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.vxlan.source_interface }}
 
-#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan_udp_port }}
+#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan.udp_port }}
```

Comment:
Please use `| arista.avd.default()` filter to report default UDP port if not set by user

Reply:
Updated
```markdown
#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan.udp_port | arista.avd.default('4789') }}
```